form-design-intro.rst

```diff
@@ -0,0 +1,49 @@
+Intro to Forms in ODK
+========================
+
+Forms used in the ODK ecosystem are XML documents following the `ODK XForms specification <https://opendatakit.github.io/xforms-spec/>`_, a subset of the `W3C XForms specification <https://www.w3.org/TR/xforms/>`_. Most ODK tools use the `ODK Javarosa <https://github.com/opendatakit/javarosa>`_ library to render forms. Form transfer between ODK components is governed by the :doc:`openrosa` API.
+
+Because of the complexity of the XForms format, we do not recommend "hand-coding" XForms. The recommended process is:
```

Comment:
We should avoid the use of quote marks as specified in the [style guide](http://docs.opendatakit.org/style-guide/#quote-marks).

Reply:
This is not that.

cf. https://en.wikipedia.org/wiki/Scare_quotes

Reply:
How about the simpler "writing XForms directly by hand"?

Reply:
(It may not be clear what "hand-coding" refers to)

Reply:
right.